python/mxnet/gluon/block.py

```diff
@@ -932,6 +932,9 @@ def _get_graph(self, *args):
 
     def _build_cache(self, *args):
         data, out = self._get_graph(*args)
+        if self._backend:
+            # To do: pass in all arguments
+            out = out.optimize_for(self._backend)
```

Comment:
also pass `self.optargs` to `optimize_for` like:
```
out = out.optimize_for(self._backend, **self.optargs)
```

Reply:
Updated. Thanks!